Dashboard.Storage/Azure/BuildUtil.cs

```diff
@@ -71,6 +71,30 @@ public List<BuildFailureEntity> GetTestCaseFailures(DateTimeOffset startDate, st
             return _buildFailureDateTable.ExecuteQuery(query).ToList();
         }
 
+        public List<string> GetViewNames(DateTimeOffset startDate)
+        {
+            var filter = FilterUtil
+                .SinceDate(ColumnNames.PartitionKey, startDate)
+                .And(FilterUtil.Column(nameof(BuildResultEntity.ViewName), null, ColumnOperator.NotEqual));
+            var query = new TableQuery<BuildResultEntity>()
```

Comment:
I think we should consider doing a `DynamicTableEntity` query here.  We are only using the `ViewName` column in the result so we can reduce the data size we are querying by just dynamically selecting that column.